elevenlabs_mcp/server.py

```
@@ -139,11 +150,47 @@ def format_diarized_transcript(transcription) -> str:
         return transcription.text
+@mcp.resource("elevenlabs://{filename}")
+def get_elevenlabs_resource(filename: str) -> Resource:
+    """
+    Resource handler for ElevenLabs generated files.
+    """
+    # Construct the file path
+    output_path = make_output_path(None, base_path)
+    file_path = output_path / filename
+
+    if not file_path.exists():
+        raise FileNotFoundError(f"Resource file not found: {filename}")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)